fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java

```diff
@@ -514,4 +516,43 @@ public static ReplicaAllocation analyzeReplicaAllocation(Map<String, String> pro
         }
         return replicaAlloc;
     }
+
+    public static DataSortInfo analyzeDataSortInfo(Map<String, String> properties, KeysType keyType,
+                                                   int keyCount, TStorageFormat storageFormat) throws AnalysisException {
+        if (properties == null || properties.isEmpty()) {
+            return new DataSortInfo(TSortType.LEXICAL, keyCount);
+        }
+        String sortMethod = "lexical";
+        if (properties.containsKey(DataSortInfo.DATA_SORT_TYPE)) {
+            sortMethod = properties.remove(DataSortInfo.DATA_SORT_TYPE);
+        }
+        TSortType sortType = TSortType.LEXICAL;
+        if (sortMethod.equalsIgnoreCase("zorder")) {
```

Comment:
use `TSortType.ZORDER.name()` instead of a raw string.

Reply:
done